django_cloud_deploy/cli/prompt.py

```diff
@@ -662,7 +633,14 @@ def prompt(cls,
         """
         console_prompt = cls._get_prompt(arguments)
         console.tell(('{} {}').format(step_prompt, console_prompt))
-        return console.getpass('Password: ')
+        while True:
+            password = console.getpass('Password: ')
```

Comment:
If this is the user's postgres password then I'd write that. Like `Postgres password: `

Reply:
Done.